projects/ngx-charts-on-fhir/src/lib/data-layer/data-layer-manager.service.ts

```diff
@@ -69,16 +72,17 @@ export class DataLayerManagerService {
    */
   retrieveAll() {
     this.loading = true;
+    this.loader$.next(true);
     merge(...this.dataLayerServices.map((service) => service.retrieve())).subscribe({
       next: (layer) =>
         this.stateSubject.next({
           ...this.stateSubject.value,
           layers: this.mergeService.merge(this.state.layers, layer),
         }),
       error: (err) => console.error(err),
-      complete: () => (this.loading = false),
+      complete: () => {this.loading = false; this.loader$.next(false);},
```

Comment:
you can probably remove the boolean `loading` property. I don't think it's used anywhere.

Reply:
Yes I was remove that boolean loading property i thought it we are using that.